lib/ansible/cli/inventory.py

```diff
@@ -327,6 +327,8 @@ def inventory_graph(self):
 
     def json_inventory(self, top):
 
+        seen = set([])
```

Comment:
Empty set is just:
```suggestion
        seen = set()
```
(no need to allocate an intermediate list)